src/styles/blocks/header.scss

```diff
@@ -38,5 +38,10 @@
     text-transform: uppercase;
     font-size: 12px;
     font-weight: 500;
+    transition-duration: 300ms;
+  }
+
+  &__link:hover {
+    color: #00acdc;
```

Comment:
Same here. Also, would be great to create variables for main values (like colors and transition time)
![image](https://user-images.githubusercontent.com/96409563/209576532-17ea9a6f-c819-43b8-a6bb-9ffb1c8cf391.png)

Reply:
done